packages/shared/src/modules/users/libs/enums/user-validation-rule.enum.ts

```diff
@@ -1,6 +1,7 @@
 const UserValidationRule = {
 	EMAIL_MAXIMUM_LENGTH: 50,
 	EMAIL_MINIMUM_LENGTH: 6,
+	LATIN_PATTERN: /^[\w!"#$%&'()*+,./:;<=>?@[\]^`{|}~-]*$/,
```

Comment:
It's not only latin and this validation looks a bit confusing and complex. It's better to avoid many regexes where it can be handled properly with one. Let's have one regext for password pattern like `/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]*$/` (replace special chars starting from @ with needed ones). This pattern will require at least one lowercase, at least one uppercase, at least one digit, and at least one special char. The message for the password will need to combine all those rules.  And for the name field, let's have a separate pattern:
```javascript
NAME_PATTERN: /^[A-Za-z\s]*$/
```
For email, I think it's already restricted and there is no need to check it.

Reply:
Oh okay, it is clearer your way

P.s. it was more like Latin + special characters 😅